lib/contextual_logger.rb

```diff
@@ -32,6 +32,15 @@ def normalize_log_level(log_level)
         end
       end
     end
+
+    def normalize_message(message)
+      case message
+      when String
+        message
+      else
+        message.inspect
```

Comment:
@jebentier I decided to keep this method simpler than `msg2str`. I didn't really want that whole exception backtrace in here... that would be really weird to log the exception without any context, so I don't want to encourage it. And honeybadger already captures backtraces itself.